hazelcast-enterprise/Dockerfile

```diff
@@ -57,7 +57,9 @@ RUN echo "Installing new packages" \
     && echo "Removing cached package data and unnecessary tools" \
     && microdnf remove zip unzip \
     && microdnf -y clean all \
-    && rm -rf ${HZ_HOME}/get-hz-ee-dist-zip.sh ${HZ_HOME}/hazelcast-enterprise-distribution.zip ${HZ_HOME}/tmp
+    && rm -rf ${HZ_HOME}/get-hz-ee-dist-zip.sh ${HZ_HOME}/hazelcast-enterprise-distribution.zip ${HZ_HOME}/tmp \
+    # unzip may not preserve permissions. Give execute permission scripts
```

Comment:
I thought it's because of quirks on Windows/WSL with building the distribution

Reply:
yes. i have suggested different comment below

Reply:
The comment is 
`Grant execute permission to scripts in order to address the issue of permissions not being accurately propagated on Windows OS`